api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java

```diff
@@ -93,7 +93,7 @@ private void installConcepts() {
         	Context.flushSession(); //Flush so that purges are not deferred until after data import
         	
         	DataImporter dataImporter = Context.getRegisteredComponent("dataImporter", DataImporter.class);
-            dataImporter.importData("Reference_Application_Concepts-24.xml");
+            dataImporter.importData("Reference_Application_Concepts-26.xml");
```

Comment:
Any reason why this is 26 instead of 25?

Reply:
sorry , it was a mistake

Reply:
corrected it